src/vs/workbench/electron-sandbox/window.ts

```diff
@@ -228,6 +228,23 @@ export class NativeWindow extends Disposable {
 			);
 		});
 
+		ipcRenderer.on('vscode:showTranslatedBuildWarning', (event: unknown, message: string) => {
+			this.notificationService.prompt(
+				Severity.Warning,
+				localize("runningTranslated", "You are running a translated build. For better performance download a native build for your machine!"),
+				[{
+					label: localize('downloadArmBuild', "Download the ARM build"),
+					run: () => {
+						if (isMacintosh) {
+							this.openerService.open('https://code.visualstudio.com/docs/?dv=osx');
+						} else {
+							this.openerService.open('https://code.visualstudio.com/download');
+						}
```

Comment:
```suggestion
                                               const quality = this.productService.quality;
                                               let stableURL = '';
                                               let insidersURL = '';
						if (isMacintosh) {
						       stableURL = 'https://code.visualstudio.com/docs/?dv=osx';
						       insidersURL = 'https://code.visualstudio.com/docs/?dv=osx&build=insiders';
						} else if (isWindows) {
						      stableURL = 'https://code.visualstudio.com/docs/?dv=win32arm64user';
						      insidersURL = 'https://code.visualstudio.com/docs/?dv=win32arm64user&build=insiders';
						}
						this.openerService.open(quality === 'stable' ? stableURL : insidersURL);
```

Reply:
Nevermind app.runningUnderARM64Translation doesn't check for linux translation.